package.json

```diff
@@ -54,11 +54,22 @@
           "arch": [
             "x64"
           ]
+        },
+        {
+          "target": "nsis",
+          "arch": [
+            "x64"
+          ]
         }
       ],
       "publisherName": "Rapid Silicon, Inc.",
+      "icon": "RPE_icon.png",
```

Comment:
Where is the icon?

Reply:
This is a placeholder so that in the future when we have an icon file for RPE we will pass that to installer via this keyword